### PR TITLE
NMS-12978: Add Prometheus Collectd Service configuration example

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-12978

### CODE REVIEW

File: docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc [hunk 1]
```diff
@@ -127,3 +127,16 @@ Here's another excerpt where we extract metrics grouped by CPU:
 ----
 
 This group definition matches metrics called 'node_cpu', groups them by the value of the `cpu` label and extracts the name of the `mode` for the name of the numeric attributes.
+
+
+== Prometheus Node Exporter Collectd Service Configuration Example
```
Comment (Bonrob2) [inline docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:132]:
```suggestion
== Prometheus Node Exporter Collectd service configuration example
```
We are starting to move towards using sentence capitalization for headings. In this case, though, we still have to capitalize Prometheus Node Exporter and Collectd, as we are using them as proper nouns in the docs.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor change. I can't comment on whether the example works, but it looks good to me.

**Comment by mfuhrmann** [comment]
@Bonrob2 The example works. If you would read the collectd logs properly, then you should be able to write this code by yourself. But having it as a copy + paste version in the docs is a little bit easier (since this one is not included in the default collectd configuration) for the users.

**Review by Bonrob2 — APPROVED** [review]
Looks great!

**Pull request merged**